doc-content/enterprise-only/project-data/new-project-proc.adoc

```diff
@@ -0,0 +1,14 @@
+[id='new_project']
+= Creating a project
+
+A project can contain data models, business processes, business rules, and forms that are specific to your project's requirements. When you create a project in {CENTRAL} it is added to the Git repository connected to {CENTRAL}.
+
+.Procedure
+. Log in to {CENTRAL} and click *Menu* -> *Design* -> *Projects*.
+. Click *Add Project*.
+. Type `anyName` in the *Name* field of the *Add Project* window and click *Add*.
```

Comment:
I think you should make it more clear that it's actually any name not "anyName"

Reply:
Done.